views/index.hbs

```diff
@@ -1,2 +1,6 @@
 <h1>{{title}}</h1>
-<p>Welcome to {{title}}</p>
+
+<ul>
+    <li><h3><a href="drones">Check the Drones</a></h3></li>
```

Comment:
Similarly here you can give a class to your `<li>` elements instead of using`<h3>`. Also you should add a '/' at the start of the `href` attribute.
```
<li class="nav-link"><a href="/drones">Check the Drones</a></li>
<li class="nav-link"><a href="/drones/create">Add New Drone</a></li>